code/game/machinery/telecomms/broadcaster.dm

```diff
@@ -158,6 +158,24 @@ var/message_delay = 0 // To make sure restarting the recentmessages list is kept
 
 
 
+/proc/Is_Bad_Connection(var/old_freq, var/new_freq) //Makes sure players cant read radios of a higher level than they are
+
+	var/old_type = 0 //0 = crew, 1 = centcomm, 2 = syndie
```

Comment:
Please change all of the `0`, `1`, and `2` to #define values (just put them right above)
```
#define CREW_RADIO_TYPE 0
#define CENTCOMM_RADIO_TYPE 1
#define SYNDICATE_RADIO_TYPE 2
/proc/Is_Bad_Connection(old_freq, new_freq)
...
```